static/app/views/settings/project/dynamicSampling/index.tsx

```diff
@@ -0,0 +1,29 @@
+import Feature from 'sentry/components/acl/feature';
+import {Project} from 'sentry/types';
+import useOrganization from 'sentry/utils/useOrganization';
+
+import ServerSideSampling from '../server-side-sampling';
+
+import {DynamicSampling} from './dynamicSampling';
+
+type Props = {
+  project: Project;
+};
+
+export default function DynamicSamplingContainer({project}: Props) {
+  const organization = useOrganization();
+
+  if (!organization.features.includes('dynamic-sampling-total-transaction-packaging')) {
+    return <ServerSideSampling project={project} />;
+  }
+
+  return (
+    <Feature
+      features={['server-side-sampling', 'server-side-sampling-ui']}
+      hookName="feature-disabled:dynamic-sampling-basic"
+      organization={organization}
+    >
+      <DynamicSampling project={project} />
```

Comment:
Should the flag check between DynamicSampling and ServerSideSampling happen here on line 26 so that they are both protected by the <Feature> component?

Reply:
I thought about it too, but we also have this "protection" in the index of the `server-side-sampling` folder